Sources/Fuzzilli/FuzzIL/JsOperations.swift

```diff
@@ -2064,6 +2064,22 @@ final class LoopContinue: JsOperation {
     }
 }
 
+final class BreakNested: JsOperation {
+    override var opcode: Opcode { .breakNested(self) }
+
+    init() {
+        super.init(numInputs: 1, attributes: [.isJump], requiredContext: [.javascript, .loop])
```

Comment:
Having the depth as input will be tricky, again because the mutator will change inputs around arbitrarily and you'll quickly get strings or objects or other stuff as depth. Instead, I'd recommend making the depth a property of this operation:
```
final class BreakNested: JsOperation {
    override var opcode: Opcode { .breakNested(self) }

    let depth: Int

    init(depth: Int) {
        ...
    }
```

Reply:
We can then also let the `OperationMutator` mutate the depth property of these.

Reply:
Your analysis has been very helpful to me. Thank you so much! I will complete the revisions as soon as possible.